### ISSUE TITLE
Window decoration option to show system window border

### ISSUE THREAD

**Issue by bluesheep100** [Jelmerro/Vieb#261]
**Checklist**
- [x] I searched the `:help` documentation using `/` for helpful information
- [x] I have read the [frequently asked questions](https://github.com/Jelmerro/Vieb/blob/master/FAQ.md)
- [x] I did not find any similar issues in the [issue tracker](https://github.com/Jelmerro/Vieb/issues)

**Addition or change**
I use dwm (dynamic window manager), and my window border doesn't show, and instead I get a small gap at the edge of the window. Browsers like Brave have an option to fix this, called "window decoration". I'm _guessing_ that this is a feature that comes from Chromium and for Vieb it simply needs to be exposed as an option in the viebrc.

TL;DR
I want a windowdecoration option so my window manager's border can show up correctly, and I can see when I have the window focused.

**Comment by Jelmerro** [comment]
This is already possible using a startup option called `window-frame`. You can run `vieb --help` for details, but in short: Either run vieb as `vieb --window-frame=true` or set `VIEB_WINDOW_FRAME=true` as an environment variable on your OS before starting Vieb.

**Comment by bluesheep100** [comment]
Alright, I've added the `VIEB_WINDOW_FRAME` variable to my xinit and it works swimmingly; thanks for that.
I hadn't thought of reading the command's help page, because I didn't think it'd be that different to the built-in help page.
Maybe it could be an idea to document some of the commandline options in other places as well?

Thanks again for the quick response, been loving this browser so far ^^

**Comment by Shamaoke** [comment]
@Jelmerro Why this isn't the default?

**Comment by Jelmerro** [comment]
Because you can do all of the things in the window bar with keyboard shortcuts or by interacting with the logo, there should be little need for the window bar from a functional perspective. You can drag the window by picking it up using the logo at the top left. You can double click this to toggle maximize and pretty much all systems have keyboard shortcuts for these and other actions like minimize as well.

Otherwise it's just mostly that it has been the default for literally years and I would want to confirm that most users want this to be enabled before changing something like this, plus that it would be a breaking change which I tend to avoid if I can. It's currently fairly trivial to change this if you want to enable it, and I have had many users list it as a plus that Vieb does not have this relatively useless window border. So that's probably why it's never been changed. It could be an FAQ entry now that I think about it, I will consider making one in the future (though it's not actually been that frequently asked, since most users I feel don't need this window bar at all).
